daft/dataframe/dataframe.py

```diff
@@ -158,6 +169,45 @@ def _result(self) -> Optional[PartitionSet]:
         else:
             return self._result_cache.value
 
+    def _explain_broadcast(self):
+        import json
+        from urllib import request
+
+        from daft.dataframe.display import MermaidFormatter
+
+        try:
+            enable_dashboard = int(os.environ.get("DAFT_DASHBOARD_ENABLE", "0"))
+        except ValueError:
+            return
+
+        if not enable_dashboard:
+            return
+
+        addr = os.environ.get("DAFT_DASHBOARD_ADDR", "localhost")
+        port = int(os.environ.get("DAFT_DASHBOARD_PORT", 3238))
```

Comment:
Why not have it all as one? `DAFT_DASHBOARD=http://localhost:3238`

Reply:
Oh ya, that does look nicer. I'll update in the next commit.

Reply:
Updated.